content/TrueCommand/Alerts/AlertManage.md

```diff
@@ -6,125 +6,179 @@ aliases:
-The *From* field allows you to customize the sender field of the email.
-For a *No-Auth* SMTP configuration, leave the password field blank.
+To properly configure SMTP email:
 
-![AlertServicesSMTPEmailptions](/images/TrueCommand/2.0/AlertServicesSMTPEmailptions.png "Alert Services: SMTP Email options")
+1. Enter values in all fields on the **SMTP Email** screen and then click **SAVE**:
+   
+   ![AlertServicesSMTPEmailptions](/images/TrueCommand/2.0/AlertServicesSMTPEmailptions.png "Alert Services: SMTP Email options")
 
-Click **Test** to verify that the configuarion is correct.
-If you did not receive a test alert email, check the values entered for accuracy.
+   * **Mailserver** for example *smtp.gmail.com*
+   * **Mailserver port** number
+   * **Auth user** email address for plain authentication, for example, *adminuser@yourmail.com*
+   * **Auth pass** password for the plain authentication; for a *No-Auth* SMTP configuration, leave the password field blank
+   * **From** is the what sends the eamil (i.e., no-reply@TrueCommand.io),or allows you to customize the sender field of the email 
 
-{{< /tab >}}
-{{< tab "PagerDuty" >}}
+  Click **Test** on the **SMTP Email** configuration screen to verify that the configuarion is correct.
+  If you did not receive a test alert email, check the values entered for accuracy.
+
+2. Click on the avatar to the right of the **Configure** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> menu and then click **Profile** from the dropdown list.
+     
+   ![EditUserProfile](/images/TrueCommand/2.1/EditUserProfile.png "Edit User Profile")
+
+3. Type the email for that user in the **Email** field and click **SAVE CHANGES**.
+   The first time you set up SMTP email a **VERIFY EMAIL** button displays below the **Email** field to the left of the **Enable 2FA** checkbox.
 
-Open the **Configure Plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> for PagerDuty.  In the
-*Authtoken* section, enter your **Service Integration Key from PagerDuty**.  If you have an active subscription with PagerDuty, this key should be available to you. 
-Enter a *Title for incident reports* if desired. Click *TEST*.
+   The system should automatically send a test email to the specified email address but if not click **VERIFY EMAIL**.  
+   A **Confirmation** dialog box displays with a field where you type or copy/paste the code sent in the email. Doing this verifies the email.
+   
+   ![SMTPSetupVerifyEmail](/images/TrueCommand/2.1/SMTPSetupVerifyEmail.png "SMTP Setup Verify Email")
+
+## Configuring PagerDuty
+
+Open the **Configure Plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> for PagerDuty. In the
+**Authtoken** section, enter your **Service Integration Key from PagerDuty**.  If you have an active subscription with PagerDuty, this key should be available to you. 
```

Comment:
Kind of an odd line break. Simple suggestion to break on the sentence.
```suggestion
Open the **Configure Plugin** <i class="material-icons" aria-hidden="true" title="Settings">settings</i> for PagerDuty.
In the **Authtoken** section, enter your **Service Integration Key from PagerDuty**. If you have an active subscription with PagerDuty, this key should be available to you. 
```

Reply:
That was an oops... I didn't catch it. Thanks